### DISCUSSION TITLE
Cannot invoke ctrl+shift+f

### DISCUSSION THREAD

**Discussion by ZhanZiyuan** [microsoft/terminal#16969] [General]
It seems that the hotkey `ctrl+shift+f` cannot be triggered while `ctrl+shift+P` works well after upgrading Windows Terminal to v1.19.10821.0 .

OS Version: Windows 10.0.19045
![Snipaste_2024-03-29_19-55-26](https://github.com/microsoft/terminal/assets/154674404/04442493-0e70-437f-ae98-1387f8fa3b17)

**Comment by zadjii-msft** [comment]
You know, we've definitely seen this before: #15598

> "Ctrl+Shift+F", which is exactly the same as the default "Simplified/Traditional Switch" shortcut key of the Microsoft Pinyin input method

Any chance you're using the Microsoft Pinyin IME?

**Reply by ZhanZiyuan** [reply]
Thank you for your reply. It's my fault to miss a few questions in my search for similar ones.
Yes, I'm using the Microsoft Pinyin IME. Maybe I should change it to English (U.S.) IME?

**Reply by ZhanZiyuan** [reply]
The hotkey `ctrl+shift+f` works well when I'm using the Microsoft English (U.S.) IME or the Microsoft Japanese IME. Thank you for your help.
